src/styles/themer/withTheme.js

```diff
@@ -0,0 +1,41 @@
+// Higher order component that force updates component on themer change
+// and passes down theme through props
+import React, { Component } from 'react'
+import themer from './index'
+import { themePropTypes } from './utils'
+
+function withTheme(InnerComponent) {
+  class Wrapped extends Component {
+    static displayName = `withTheme(${InnerComponent.displayName})`
+
+    static propTypes = {
+      snacksTheme: themePropTypes
+    }
+
+    componentDidMount() {
+      this.unsubscribe = themer.subscribe(this.onThemeChange)
+    }
+
+    componentWillUnmount() {
+      this.unsubscribe()
+    }
+
+    onThemeChange() {
+      this.forceUpdate()
+    }
+
+    render() {
+      return (
+        <InnerComponent
+          ref={(node) => this.wrapped = node}
```

Comment:
Why do we need the ref here?

Reply:
+1 , unused as is, but maybe in prep for hoisting statics?

Reply:
oops, yep. Was going to do hoisting and decided against it. Will remove for now.